splink/cluster_metrics.py

```diff
@@ -137,3 +138,27 @@ def _size_density_centralisation_sql(
     sqls.append(sql)
 
     return sqls
+
+
+@dataclass
+class GraphMetricsResults:
+    nodes: SplinkDataFrame
+    edges: SplinkDataFrame
+    clusters: SplinkDataFrame
+
+    def __repr__(self):
+        return """
+A data class of Splink dataframes containing metrics for nodes, edges and clusters.
+
+Access dataframes via attributes:
+`compute_graph_metrics.nodes` for node metrics,
+`compute_graph_metrics.edges` for edge metrics and
```

Comment:
Just personal preference but I always find it weird having multi-line strings that mess with the indentation like this - I think that using implicit continuation and explicit newline `"\n"` characters reads clearer (see for example `SplinkDataFrame.__repr__`).

But not a big deal at all, so am fine if I'm in the minority on this

Reply:
This is really useful feedback! It looks icky to me too so have updated 👍